lib/src/supertokens.dart

```diff
@@ -69,4 +94,91 @@ class SuperTokens {
     String? idRefreshToken = await IdRefreshToken.getToken();
     return idRefreshToken != null;
   }
+
+  static Future<void> signOut(Function(Exception?) completionHandler) async {
+    if (!(await doesSessionExist())) {
+      SuperTokens.config!.eventHandler(Eventype.SIGN_OUT);
+      completionHandler(null);
+      return;
+    }
+
+    Uri uri;
+    try {
+      uri = Uri.parse(SuperTokens.signOutUrl);
+    } catch (e) {
+      completionHandler(SuperTokensException(
+          "Please provide a valid apiDomain and apiBasePath"));
+      return;
+    }
+
+    http.Request signOut = http.Request('post', uri);
```

Comment:
We want to use the supertokens http client for sign out so that interception is done